app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    list_files = command.split()
+    if (len(list_files) == 3
+            and list_files[0] == "mv"
+            and list_files[1] != list_files[2]):
+        depth = list_files[2].split("/")
```

Comment:
Don't use `split("/")` for splitting path to file. Python has a specific function for this - `os.path.split`